store/mockstore/mocktikv/mock.go

```diff
@@ -18,8 +18,8 @@ import (
 	"github.com/pingcap/pd/pd-client"
 )
 
-// NewTestClient creates a TiKV client and PD client from options.
-func NewTestClient(cluster *Cluster, mvccStore MVCCStore, path string) (*RPCClient, pd.Client, error) {
+// NewPDAndTiKVClient creates a TiKV client and PD client from options.
+func NewPDAndTiKVClient(cluster *Cluster, mvccStore MVCCStore, path string) (*RPCClient, pd.Client, error) {
```

Comment:
NewTiKVAndPDClient is better. Because it has the same order with the return values.

Reply:
Gotcha. Working on it right away.